crates/opactl/src/main.rs

```diff
@@ -312,13 +315,13 @@ async fn dispatch_args<
         Some(("set-policy", matches)) => {
             let root_key: SigningKey = load_key_from_match("root-key", matches).into();
             let transactor_key: SigningKey = load_key_from_match("transactor-key", matches).into();
-            let policy: &PathBuf = matches.get_one("policy").unwrap();
-            let mut policy_file = File::open(policy)?;
-            let mut policy = vec![];
-            policy_file.read_to_end(&mut policy)?;
+            let policy: &String = matches.get_one("policy").unwrap();
```

Comment:
We can currently get the policy into the Helm Chart via the built default policies that are built into `opactl`. Can we not keep that option and have another `--from-url` option? Otherwise, without a policy bundle at an address, this breaks the set-policy support in the Helm Chart.

Reply:
to @suchapalaver's point, and the title of the PR,  this should also take a file path, and `reqwest` apparently does not, nor even a `file://`  url

Reply:
I'll stick a protocol switch in front

Reply:
Building and testing with the chart.